docs/snippets/vue/button-story-click-handler-simple-docs.vue3.js.mdx

```diff
@@ -0,0 +1,14 @@
+```js
+// Button.stories.js
+
+export const Text = ({ label, onClick }) => ({
```

Comment:
```suggestion
export const Text = ({ label }) => ({
```

Reply:
@phated fixed with the latest commit. Thanks for the clarification.